### PR TITLE
Feature/fix missing documentation

### PR DESCRIPTION
**Pull request by grantkemp**
improvements to Example Readme to make sure my mistakes aren't made by others

### CODE REVIEW

**Comment by thomasvl** [comment]
Rather than putting this in the one example directory, why not make it generic and put it at the top of the Examples directory as the advice should apply to all of them?

File: Examples/README.MD [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Then, add another step after the git pull.
+
+```
+git pull ...
+git submodule update --recursive
+```
+
+##Requirements to use
+These example apps need an oAuth Client key, which you can obtain from the[Google Developer Console](https://console.developers.google.com/).
```
Comment (thomasvl) [inline Examples/README.MD]: Probably want a space after "the" before the link to the console.

**Comment by grantkemp** [comment]
Just wanted to add one last item to the Readme to make it clear for novice users.  Otherwise the warning is hidden in the comments and might get missed.

**Comment by thomasvl** [comment]
Sorry, you might need to spin a new pull request for that.  As I think it arrived after I'd done a [squashing merge](https://github.com/google/google-api-objectivec-client-for-rest/commit/61b5dcdc3b0a2d2d4ee89d7fadbdbc29fcdffbcc)

**Comment by grantkemp** [comment]
Its ok, lets let it go. 
 I am going to try  to create an additional example for myself using the Google Sign In Flow ( [mentioned in the wiki ](https://github.com/google/google-api-objectivec-client-for-rest/wiki#authentication-and-authorization) assuming thats possible.  It might be good to show a different approach to leverage this library.

**Pull request merged**